default/empire_statistics.txt

```diff
@@ -1,3 +1,11 @@
+
+Statistic name = "MILITARY_STRENGTH_STAT" value =
+    Sum value = (LocalCandidate.Attack * (LocalCandidate.Structure + 2 * LocalCandidate.MaxShield))
```

Comment:
The same result could probably have been scripted by using 2 Sum statistics added together instead of a single Sum with an add in the value. But this is easier to read.